tests/cms/views/organizations/test_organization_form.py

```diff
@@ -0,0 +1,140 @@
+
+if TYPE_CHECKING:
+    from django.test.client import Client
+    from pytest_django.fixtures import SettingsWrapper
+
+import pytest
+from django.test.client import Client
+from django.urls import reverse
+
+from integreat_cms.cms.models import MediaFile, Organization, Page, POI, Region
+from tests.conftest import ANONYMOUS, MANAGEMENT, STAFF_ROLES
+
+
+@pytest.mark.django_db
+def test_poi_form_shows_no_contents(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+    settings: SettingsWrapper,
+) -> None:
+    """
+    Test that no contents are shown in the organization form if it does not have any content assinged
+    """
+    client, role = login_role_user
+
+    # Test for english messages
+    settings.LANGUAGE_CODE = "en"
+
+    new_organization = Organization.objects.create(
+        name="New Organization",
+        slug="new-organization",
+        icon=MediaFile.objects.filter(id=1).first(),
+        region=Region.objects.filter(id=1).first(),
+        website="https://integreat-app.de/",
+    )
+
+    edit_organization = reverse(
+        "edit_organization",
+        kwargs={
+            "slug": new_organization.slug,
+            "region_slug": new_organization.region.slug,
+        },
+    )
+    response = client.get(edit_organization)
+
+    if role in STAFF_ROLES + [MANAGEMENT]:
+        assert (
+            "This organization currently has no maintained pages."
+            in response.content.decode("utf-8")
+        )
+        assert (
+            "This organization currently has no maintained locations."
+            in response.content.decode("utf-8")
+        )
+
+    elif role == ANONYMOUS:
+        assert response.status_code == 302
+        assert (
+            response.headers.get("location")
+            == f"{settings.LOGIN_URL}?next={edit_organization}"
+        )
+    else:
+        assert response.status_code == 403
+
+
+@pytest.mark.django_db
+def test_poi_form_shows_associated_contents(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+    settings: SettingsWrapper,
+) -> None:
+    """
+    Test Organization in Augsburg has two pages and one location.
+    They must be shown in the form.
+    """
+    # Choose an organitation which has a page and a poi assigned
+    ORGANIZAION_ID = 1
+
+    # Choose a page and a POI
+    #   - 1) that do not belong to the organization chosen above
+    # and
+    #   - 2) whose titles are not contained in the titles of pages and organizations that belong to the organization
+    NON_ORGANIZATION_PAGE_ID = 5
+    NON_ORGANIZATION_POI_ID = 6
+
+    client, role = login_role_user
+
+    # Test for english messages
+    settings.LANGUAGE_CODE = "en"
+
+    organization = Organization.objects.filter(id=ORGANIZAION_ID).first()
+    organization_pages = list(organization.pages.all())
+    organization_pois = list(organization.pois.all())
+
+    region = organization.region
+
+    # Make sure this organization has pages and contents
```

Comment:
```suggestion
```
Nit: I think your code is clean and self-explanatory enough to delete this line :)